packages/logger/src/lib/logger-named.ts

```diff
@@ -0,0 +1,41 @@
+import { LogConsumer } from './model/log-consumer';
+import { LogLevel } from './model/log-level';
+import { Logger } from './model/logger';
+
+export interface NamedLogger {
+  logLevel: LogLevel;
+
+  debug(...messages: unknown[]): void;
+
+  info(...messages: unknown[]): void;
+
+  warn(...messages: unknown[]): void;
+
+  error(...messages: unknown[]): void;
+
+  addConsumer(consumer: LogConsumer): void;
+}
+
+export function createNamedLogger(name: string, logger: Logger): NamedLogger {
```

Comment:
logger should fall back to the global logger instance

Reply:
Fixed.